src/infrasys/time_series_models.py

```diff
@@ -257,12 +257,12 @@ def deserialize_from_strings(cls, values: dict[str, Any]) -> dict[str, Any]:
         return values
 
 
-class TimeSeriesMetadata(InfraSysBaseModel, abc.ABC):
+class TimeSeriesMetadata(InfraSysBaseModelWithIdentifers, abc.ABC):
```

Comment:
Is there an expectation that IS.jl and infrasys will be able to deserialize each other's database files without special handling? I see at least two things that will prevent it from working:
1. They encode special serialization keys into the metadata-as-JSON object. We could unify this if we change both sides to follow a common specification.
2. This uses `variable_name` while IS.jl uses `name`. Even though you made the column name match, that field is encoded in the metadata-as-JSON object.

Reply:
I think it makes sense for me they can deserialize each other w/o. special handling. Is there any other fields that we should be worried other than variable name?

Reply:
Proposal:
1. Add `scaling_factor_multiplier TEXT NULL` as a column to the table.
2. Add `serialization_info TEXT NOT NULL` as a column to the table. This contains the fields needed by Sienna to deserialize the metadata into an instance of a Julia structure.
3. Remove `time_series_category` from the table.
4. Don't create the second metadata table during serialization.

Example contents of the serialization_info column:
```
"{\"internal\":{\"uuid\":{\"value\":\"6d326d0d-7c7b-4d9a-9f8d-bf2451ef34f3\"},\"ext\":null,\"units_info\":null},\"scaling_factor_multiplier\":{\"__metadata__\":{\"function\":\"get_max_active_power\",\"module\":\"PowerSystems\"}},\"__metadata__\":{\"module\":\"InfrastructureSystems\",\"type\":\"SingleTimeSeriesMetadata\"}}"
```